src/models/daily-auths-report-data.ts

```diff
@@ -1,4 +1,4 @@
-import { utcDays } from "d3-time";
+import { utcDays, timeDay } from "d3-time";
```

Comment:
I think `timeDay` may be local timezone, but to keep things consistent we should use `utcDay`
```suggestion
import { utcDays, utcDay } from "d3-time";
```